ccd-definition/AuthorisationCaseField.json

```diff
@@ -660,78 +716,253 @@
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "CaseFieldID": "standardDirectionsLabel",
-    "UserRole": "caseworker-publiclaw-courtadmin",
+    "CaseFieldID": "standardDirectionsDocument",
```

Comment:
Diff is obfuscated a bit due to the fact I moved fields around to group fields by role. Now we have block of solicitor fields, blocks of court admin fields and so on. Generally speaking gatekeeper should have roughly same access as court admin.